src/test/java/samples/android/nativeapp/apidemos/screens/ViewControlsScreen.java

```diff
@@ -11,15 +15,41 @@ public ViewControlsScreen() {
         super(By.id("android:id/content"), "View/Controls");
     }
 
-    public IRadioButton getRadioButton(int number){
+    public IRadioButton getRadioButton(int number) {
         return getElementFactory().getRadioButton(
                 MobileBy.AccessibilityId(String.format("RadioButton %d", number)),
                 String.valueOf(number));
     }
 
-    public ICheckBox getCheckBox(int number){
+    public ICheckBox getCheckBox(int number) {
         return getElementFactory().getCheckBox(
                 MobileBy.AccessibilityId(String.format("Checkbox %d", number)),
                 String.valueOf(number));
     }
+
+    private ILabel lblAllInsideScrollView() {
+        return getElementFactory().getLabel(
```

Comment:
this element is not depend from any parameters, please define it as private final field.
Same for elements below